packages/@aws-cdk/custom-resources/lib/aws-custom-resource/runtime/index.ts

```diff
@@ -67,6 +67,10 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
         region: call.region,
       });
 
+      if (event.RequestType === 'Delete' && call.physicalResourceIdPath) {
+        call.parameters[call.physicalResourceIdPath] = physicalResourceId;
+      }
```

Comment:
If I understand it correctly, you want to be able to use the "current" physical id somewhere in you API call for the delete event?

The `physicalResourceIdPath` works the other way around: extracts the physical id from the response to return it to CloudFormation. You didn't update the documentation for this prop, so only you would know about this behavior here.

How about having some sort of tag (`{{PHYSICAL_ID}}`?) that could be used in the `parameters` prop and that would get replaced in the Lambda before the API call?

Can you maybe detail your use case?

@eladb opinion on this?

Reply:
I'll add an example inline here, and will update the PR tomorrow morning.

When I create a `DatasetGroup` resource using `AwsCustomResource` like this:
```
new AwsCustomResource(this, 'DatasetGroup', {
  onCreate: {
    service: 'Personalize',
    action: 'createDatasetGroup',
    parameters: {
      name: 'TestDatasetGroup'
    },
    physicalResourceIdPath: 'datasetGroupArn'
  },
  onDelete: {
    service: 'Personalize',
    action: 'deleteDatasetGroup'
  }
})
```

The resource is created as expected.
But, when I tear down the stack, the call parameters for `deleteDatasetGroup` don't contain `datasetGroupArn`, the only parameter that's needed for the call. 

Same issues exists for other resources on Personalize & Forecast.

Reply:
Wouldn't the following work here?
```ts
const groupName = 'TestDatasetGroup';
new AwsCustomResource(this, 'CreateDatasetGroup', {
  onCreate: {
    service: 'Personalize',
    action: 'createDatasetGroup',
    parameters: {
      name: groupName
    },
    physicalResourceIdPath: 'datasetGroupArn'
  },
  onDelete: {
    service: 'Personalize',
    action: 'deleteDatasetGroup',
    parameters: { // arn:aws:personalize:<region>:<account>>:dataset-group/TestDatasetGroup
      datasetGroupArn: cdk.Stack.of(this).formatArn({
        service: 'personalize',
        resource: 'dataset-group',
        resourceName: groupName
      })
    }
  }
});
```

Reply:
Thanks @jogold, I'll try this out today.
If that works as expected, I can use that for now. 

It'd be good to know what @eladb prefers here, before I send out an update on the PR (or close it).

Cheers

Reply:
I am with @jogold